config/prow/autobump-config/prow-job-autobump-config.yaml

```diff
@@ -28,3 +28,8 @@ prefixes:
     repo: "https://github.com/kubernetes/test-infra"
     summarise: false
     consistentImages: false
+  - name: "Prow - ci-infra"
```

Comment:
I suspect, that bumping the jobs will be tackled by the other autobump config already (as it includes all config paths).
Maybe we can drop this one here.

Reply:
The prefix is different though. Without this entry autobumper won't check for images in "eu.gcr.io/gardener-project/ci-infra/" registry